From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: [patch] fix missing includes

When built with gcc-4.3.2, cgicc-3.2.7 cannot be compiled because of a
missing include:

dns.cpp:128: error: 'memcpy' was not declared in this scope
dns.cpp:160: error: 'memcpy' was not declared in this scope
make[2]: *** [dns.o] Error 1

Fix it.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
Upstream-status: https://savannah.gnu.org/bugs/?25211

---
 demo/dns.cpp |    1 +
 1 file changed, 1 insertion(+)

Index: cgicc-3.2.7/demo/dns.cpp
===================================================================
--- cgicc-3.2.7.orig/demo/dns.cpp
+++ cgicc-3.2.7/demo/dns.cpp
@@ -30,6 +30,7 @@
  */
 
 #include <cstdlib>
+#include <cstring>
 #include <new>
 #include <vector>
 #include <stdexcept>
